### PR TITLE
README changes

### PR DESCRIPTION
**Pull request by TacoTheDank**
From https://github.com/cgeo/cgeo/pull/9531

Ported README changes from upstream mapsme/api-android.

Fixed a bunch of grammatical issues in regular README and cleaned up its layout. Fixed a bunch of older links.

Also, added the cgeo icon to the README for a little bit of flavor :)

Check out https://github.com/TacoTheDank/cgeo/tree/readme-changes to see how it looks.

### CODE REVIEW

**Comment by bekuno** [comment]
ok to test

**Comment by bekuno** [comment]
@TacoTheDank 
Please make 2 separate PR here. 
On the one side you ported README changes from upstream mapsme/api-android. 
But the other is a refresh of the c:geo README. Perhaps this could lead to discussions.

File: README.md [hunk 1]
```diff
@@ -114,5 +133,5 @@ c:geo is distributed under [Apache License, Version 2.0](https://www.apache.org/
 - Twitter: https://twitter.com/android_gc
 - Facebook: https://www.facebook.com/android.geocaching
 - Google Play: https://play.google.com/store/apps/details?id=cgeo.geocaching
-- Live status: https://status.cgeo.org/
-- Developer chat: [#cgeo on freenode.net](https://webchat.freenode.net/?channels=%23cgeo)
+- Live status: https://www.cgeo.org/status.html
```
Comment (bekuno) [inline README.md:136]: @Lineflyer We should stay with `https://status.cgeo.org/`, isn't it?

Comment (Lineflyer) [reply README.md:136]: Currently it is the same, but who knows if the redirect changes in future. So better stay with status.cgeo.org for now.

**Comment by moving-bits** [comment]
@bekuno 
> Please make 2 separate PR here.
> On the one side you ported README changes from upstream mapsme/api-android.
> But the other is a refresh of the c:geo README. Perhaps this could lead to discussions.

The PR consists of three commits. If content separation is ok for you, we can just merge those "as is", without squashing the commits into one.

File: README.md (hunk 1 above)
Comment (TacoTheDank) [reply README.md:136]: 👍

**Review by fm-sys — APPROVED** [review]
Looks good IMHO

File: README.md [hunk 2]
```diff
@@ -1,111 +1,130 @@
...
-For example we have a collection of [urgent issues](https://github.com/cgeo/cgeo/projects/6) and a list of [beginner topics](https://github.com/cgeo/cgeo/projects/7) which collects issues, which might be suitable for your first contribution.
+You can also take a look at the [project page](https://github.com/cgeo/cgeo/projects) of our repository.
+For example, we have a collection of [urgent issues](https://github.com/cgeo/cgeo/projects/6) and a list of [beginner topics](https://github.com/cgeo/cgeo/projects/7).
+They collect issues that might be suitable for your first contribution.
 
 ## Project status
 
 [![Build Status](https://ci.cgeo.org/job/cgeo%20continuous%20integration/badge/icon)](https://ci.cgeo.org/job/cgeo%20continuous%20integration/)<br>
-[![Codacy Badge](https://api.codacy.com/project/badge/grade/3256314c8ba8457b9639bd2d4f4e7c91)](https://www.codacy.com/app/cgeo/cgeo)<br>
+[![Codacy Badge](https://api.codacy.com/project/badge/grade/3256314c8ba8457b9639bd2d4f4e7c91)](https://app.codacy.com/manual/cgeo/cgeo/)<br>
```
Comment (fm-sys) [inline README.md:24]: The new link seems like it isn't working, while the old one still does. Can we stay with the old link?

File: mapswithme-api/README.MD [hunk 3]
```diff
@@ -57,11 +60,11 @@ For multiple points use [MWMPoint][linkPointClass] class:
 
     void showMultiplePoints(List<SomeDomainObject> list)
     {
-      // Convert objects to MMWPoints
+      // Convert objects to MWMPoints
       final MWMPoint[] points = new MWMPoint[list.length];
       for (int i = 0; i < list.size; i++)
       {
-        // Get lat, lon, and name from object and assign it to new MMWPoint
+        // Get lat, lon, and name from object and assign it to new MWMPoint
```
Comment (bekuno) [inline mapswithme-api/README.MD:67]: Tanks for fixing this typo - now the source needs also a PR for it :-)

**Comment by bekuno** [comment]
retest this please

**Comment by fm-sys** [comment]
Tests are failing as GC3FJ5F is currently disabled...

**Comment by bekuno** [comment]
> The PR consists of three commits. If content separation is ok for you, we can just merge those "as is", without squashing the commits into one.

The change to mapsme is fine.

But I don`t know if there is need for discussion of the c:geo readme changes.
I can live with one PR.

By the way, adding the c:geo icon is a very good idea, imho.

File: README.md (hunk 2 above)
Comment (TacoTheDank) [reply README.md]: Sure

**Review by moving-bits — APPROVED** [review]
LGTM, thanks @TacoTheDank for your contribution!

**Pull request merged**